### PR TITLE
Translations for task 179 and 183 added

### PR DESCRIPTION
**Pull request by ant3k96**
Added translations for task 179 and 183 regarding release 3.1.

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19/Locales/ar.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
...
   "CONSENT_THREE_PARAGRAPH_SECTION_ONE": "لاستخدام Smittestopp، يجب السماح للتطبيق بإرسال وتخزين البيانات الموضحة أعلاه. ",
   "CONSENT_THREE_PARAGRAPH_SECTION_TWO": "عندما توافق على استخدام Smittestopp، سوف تكون قادرا على تلقي إشعار بأنك كنت بالقرب من شخص ثبتت إصابتە بـ کوفید-19. وإذا أصبت نفسك بالعدوی، فيمكنك اختيار ما إذا كنت ترغب في إرسال إشعار حول ذلك عبر Smittestopp. لن يتلقى أحد إشعارًا إذا أصبت بالعدوى حتى توافق أنت على إرسال الإشعارات. سيطلب منك الموافقة على ذلك.",
-  "CONSENT_FOUR_PARAGRAPH": "المعهد النرويجي للصحة العامة هو المسؤول عن Smittestopp. ويمكن الاتصال بالمعهد النرويجي للصحة العامة على العنوان التالي: <br><br>Folkehelseinstituttet <br>Postboks 222 Skøyen <br>0213 oslo <br> البريد الإلکتروني:  folkehelseinstituttet@fhi.no<br> رقم الهاتف: +4721077000",
+  "CONSENT_FOUR_PARAGRAPH": "المعهد النرويجي للصحة العامة هو المسؤول عن Smittestopp. ويمكن الاتصال بالمعهد النرويجي للصحة العامة على العنوان التالي: <br><br>Folkehelseinstituttet <br>Postboks 222 Skøyen <br>0213 oslo <br> البريد الإلکتروني:  folkehelseinstituttet@fhi.no<br> +47 رقم الهاتف:  21077000  ",
```
Comment (Michael-Olesen) [inline NDB.Covid19/NDB.Covid19/Locales/ar.json:44]: Why this change?

Comment (ant3k96) [reply NDB.Covid19/NDB.Covid19/Locales/ar.json]: I've changed it back, I didn't notice that while copying from excel file

**Review by Michael-Olesen — APPROVED** [review]
